arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -16,6 +20,41 @@
 PYJAGS_PRIOR_DICT = {"b": np.random.randn(3, 10, 3), "int": np.random.randn(1, 10, 3)}
 
 
+EIGHT_SCHOOL_DATA = {
+    "J": 8,
+    "y": np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0]),
+    "sigma": np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0]),
+}
+
+EIGHT_SCHOOL_PRIOR_MODEL_CODE = """
```

Comment:
This is a great PR so please dont take any of these comments as major criticism. Just as someone that is trying to understand this code and keep it working and alive as much as possible.

This eight schools code is only used in one place. It would be helpful if it were moved to `jags_prior_model` so its clear its not used anywhere else. The global makes it seem like its reused

Reply:
Ok, I moved both the prior and the posterior model code to the functions where they are used.